src/util/backAlertIncompleteSection.js

```diff
@@ -0,0 +1,15 @@
+import {Alert} from '@indec/react-native-commons/src/util';
+
+const backAlertIncompleteSection = (onPrevious, section) => Alert.alert(
```

Comment:
rename to `alertIncompleteSectionOnBack`

Reply:
Listo